src/components/tab-title/tab-title.scss

```diff
@@ -7,7 +7,40 @@
 :host([layout="center"]) {
   @apply my-0 mx-5 text-center;
   flex-basis: theme("spacing.48");
-  margin: auto;
+  .content {
+    @apply m-auto;
+  }
+}
+
+// center the text visually and not affected by the x button so as to avoid moving when on or off
+:host([layout="center"][closable]) {
+  .content {
+    padding-inline-start: 32px; //28px button width + 0.25rem padding
+  }
```

Comment:
Can I get guidance on how to implement tokens here? 
eg. `padding-inline-start: var(--core-sizing-11);`
This doesn't seem to take any effect.

Reply:
@Elijbet this is good for the release. Can you open an issue to address this by using our design tokens/tailwind for this padding? Ideally both the button width and padding should be tied to variables.